content/v2/billing-charges.markdown

```diff
@@ -0,0 +1,88 @@
+---
+title: Billing Charges API | DNSimple API v2
+excerpt: This page documents the DNSimple billing charges API v2 endpoint.
+---
+
+# Billing Charges API
+
+* TOC
+{:toc}
+
+## Charges attributes {#charges-attributes}
+
+Please refer to the definition of the `Charge` data type in [our OpenAPI documentation](/v2/openapi.yml).
+
+## List charges {#listCharges}
+
+<%= render "v2-preview" %>
+
+Lists the billing charges for the account.
+
+~~~
+GET /:account/billing/charges
+~~~
+
+### Parameters
+
+| Name       | Type      | Description    |
+| ---------- | --------- | -------------- |
+| `:account` | `integer` | The account ID |
+
+### Filters
+
+| Name          | Description                                 |
+| ------------- | ------------------------------------------- |
+| `:start_date` | Only include results after the given date.  |
```

Comment:
We should provide an example of the format.

Reply:
Actually it's in the examples below, so it may be good. Although I would still add a format info.

Reply:
Format information added in https://github.com/dnsimple/dnsimple-developer/pull/536/commits/1234b288936a4c74816ccfb65c360e7e65a3b32f